packages/stylelint-plugin-ckeditor5-rules/README.md

```diff
@@ -0,0 +1,55 @@
+CKEditor 5 Stylelint Plugins
+=========================
+
+TBA badges
+
+A set of plugins used by the [CKEditor 5](https://ckeditor.com) team for [Stylelint](https://stylelint.io/)
+
+By default this plugin is added to our [`stylelint-config-ckeditor5`](https://www.npmjs.com/package/stylelint-config-ckeditor5) preset.
+
+## Usage
+
+```
+npm i --save-dev stylelint-plugin-ckeditor5-rules
+```
+
+Configure Stylelint with a `.stylelintrc` file using the following contents:
+
+```json
+{
+	// ...
+	"plugins": [
+		// ...
+		"./lib/rules/license-header.js" // Add the plugin to the linter.
+	],
+	"rules": {
+		"ckeditor5-plugin/license-header": [ true, {
+			"headerContent": [
+				" * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.",
+				" * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license"
+			]
```

Comment:
```suggestion
			"headerContent": [
				"/*",
				" * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.",
				" * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license",
				" */"
			]
```

`eslint` requires full comment content (including the comment symbols). Let's follow this way.